web-calculator.html

```diff
@@ -0,0 +1,25 @@
+<html>
+
+<head>
+    <h1>
+        Enter one number into box A and one number into box B. Then chose your operation.
+    </h1>
+
+
+    <script src='web-calculator.js'></script>
+    <link rel='stylesheet' href="web-calculator.css">
+    </link>
+</head>
+
+<body>
+    <span class="addIcon">A</span>
+    <input id='a' type='number' />
+    <span class="addIcon">B</span>
+    <input id='b' type='number' />
+    <button type='button' class='add' onclick='ADD()'>Add</button>
```

Comment:
Fun trick, you can set the outline property to remove that default blue highlight on button click.

```CSS
button {
  outline: none;
}
```